src/components/store/Navbar/style.module.scss

```
@@ -0,0 +1,41 @@
+@use 'src/styles/vars.scss' as vars;
+
+.navbar {
+  align-items: center;
+  display: flex;
+  height: 24px;
+
+  @media (max-width: vars.$breakpoint-sm) {
+    font-size: 0.75em;
+  }
+
+  .back {
+    align-items: center;
+    background: none;
+    color: var(--theme-text-on-background-1);
+    display: flex;
+    font: inherit;
+    font-weight: bold;
+    gap: 8px;
+
+    @media (max-width: vars.$breakpoint-sm) {
```

```suggestion
    @media screen and (max-width: vars.$breakpoint-sm) {
```

nit but good practice to follow for media queries. 
`@ media print` would be css that applies for printing a webpage for example